internal/db/query.sql

```diff
@@ -0,0 +1,28 @@
+-- name: GetCronJob :one
+SELECT * FROM crons
+WHERE id = ? LIMIT 1;
+
+-- name: ListCronJobs :many
+SELECT * FROM crons
+ORDER BY name;
+
+-- name: CreateCronJob :one
+INSERT INTO crons (
+  name, expression, script
+) VALUES (
+  ?, ?, ?
+)
+RETURNING *;
+
+-- name: UpdateCronJob :exec
+UPDATE crons
+set
+    name = ?,
+    expression = ?,
+    script = ?
```

Comment:
if some of these are missing It'll override the original value (zero value of string is `""`)